spur_2dnav/launch/includes/move_base.launch.xml

```diff
@@ -0,0 +1,39 @@
+<!-- 
+    ROS navigation stack with velocity smoother and safety (reactive) controller
+-->
+<launch>
+  <include file="$(find turtlebot_navigation)/launch/includes/velocity_smoother.launch.xml"/>
+  <include file="$(find turtlebot_navigation)/launch/includes/safety_controller.launch.xml"/>
+  
+  <arg name="odom_frame_id"   default="odom"/>
+  <arg name="base_frame_id"   default="base_footprint"/>
+  <arg name="global_frame_id" default="map"/>
+  <arg name="odom_topic" default="/robot_pose_ekf/odom_combined" />
```

Comment:
This actually doesn't make sense causing the following error (also discussed often, [e.g. this one](http://answers.ros.org/question/9852/map-compatability-issue-with-move_base-and-gmapping/)). But if I set it as `/odom`, robot keeps rotating infinitely and almost never reaches the goal.

```
[ERROR] [1304145896.391400959, 492.700000000]: Client [/move_base_node] wants topic /odom to have datatype/md5sum [nav_msgs/Odometry/cd5e73d190d741a2f92e81eda573aca7], but our version has [geometry_msgs/PoseWithCovarianceStamped/953b798c0f514ff060a53a3498ce6246]. Dropping connection.
```

So what's happening with the line above is that odometry info is most likely not used by `move_base` I think, when 2D nav works "fine" (not yet perfect for sure).